### PR TITLE
Decrease command and wind speed subscriber queue size to 1

### PR DESCRIPTION
**Pull request by sqrt**
as discussed with @markusachtelik

### CODE REVIEW

**Comment by markusachtelik** [comment]
add to whitelist.

**Comment by markusachtelik** [comment]
only OS X fails because of some missing pcl stuff. Ok to merge from my side, let's wait for @ffurrer's ok.

File: rotors_gazebo_plugins/src/gazebo_motor_model.cpp [hunk 1]
```diff
@@ -108,8 +108,8 @@ void GazeboMotorModel::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf) {
   // simulation iteration.
   updateConnection_ = event::Events::ConnectWorldUpdateBegin(boost::bind(&GazeboMotorModel::OnUpdate, this, _1));
 
-  command_sub_ = node_handle_->subscribe(command_sub_topic_, 1000, &GazeboMotorModel::VelocityCallback, this);
-  wind_speed_sub_ = node_handle_->subscribe(wind_speed_sub_topic_, 1000, &GazeboMotorModel::WindSpeedCallback, this);
+  command_sub_ = node_handle_->subscribe(command_sub_topic_, 1, &GazeboMotorModel::VelocityCallback, this); // PR machen
```
Comment (ffurrer) [inline rotors_gazebo_plugins/src/gazebo_motor_model.cpp]: What is the comment for?

Comment (markusachtelik) [inline rotors_gazebo_plugins/src/gazebo_motor_model.cpp]: I guess this can be removed @sqrt  ;)

**Pull request merged**